### PR TITLE
docs: update layout content in form components

### PR DESCRIPTION
**Pull request by Shijir**
Signed-off-by: stsogoo <stsogoo@vmware.com>

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by hippee-lee — APPROVED** [review]

File: apps/website/core-components/file/README.md [hunk 1]
```diff
@@ -53,3 +53,69 @@ Use a file input field when the user needs to choose one or more files from thei
...
+
+::: component-section-level-one-title
+
+## Layouts
+
+:::
+
+::: component-section-level-one
+
+Files are available in vertical, horizontal, and compact layout. See more layout guidelines on the [form page](/web-components/form/).
```
Comment (mathisscott) [inline apps/website/core-components/file/README.md]: `File inputs` instead of `Files`?

File: apps/website/core-components/input/README.md [hunk 2]
```diff
@@ -71,42 +71,48 @@ Use a select:
 
 ::: component-section-level-one
 
-Form groups can have one of three layouts. Vertical, horizontal or compact. This layout will apply to all control inputs in the group.
+Inputs are available in vertical, horizontal, compact, and controled-width layout. Controled-width layout is an additional option for setting the width in cases where the control may be unattractive or less usable if the underline is allowed to fill the full width of its container. Example: The text the user will input is of a known character count or within a small range. See more layout guidelines on the [form page](/web-components/form/).
```
Comment (mathisscott) [inline apps/website/core-components/input/README.md]: `controlled-width` not `controled-width`

File: apps/website/core-components/range/README.md [hunk 3]
```diff
@@ -62,3 +62,69 @@ This is an imprecise input control that should only be used when the exact value
...
+
+::: component-section-level-one-title
+
+## Layouts
+
+:::
+
+::: component-section-level-one
+
+Range are available in vertical, horizontal, and compact layout. See more layout guidelines on the [form page](/web-components/form/).
```
Comment (mathisscott) [inline apps/website/core-components/range/README.md]: `Range inputs` not `range`

File: apps/website/core-components/search/README.md [hunk 4]
```diff
@@ -53,3 +53,69 @@ Use a search input when user is expected to perform a search.
...
+
+::: component-section-level-one-title
+
+## Layouts
+
+:::
+
+::: component-section-level-one
+
+Search is available in vertical, horizontal, and compact layout. Horizontal layouts are preferred when it is often used on its own in a horizontal container such as an application header. See more layout guidelines on the [form page](/web-components/form/).
```
Comment (mathisscott) [inline apps/website/core-components/search/README.md]: `Search inputs are` not `Search is`

File: apps/website/core-components/select/README.md [hunk 5]
```diff
@@ -77,3 +77,87 @@ A common strategy is to combine an input field with a select box so that a user
...
+
+::: component-section-level-one-title
+
+## Layouts
+
+:::
+
+::: component-section-level-one
+
+Select is available in vertical, horizontal, compact, and multiple layout. The select multiple option enables to user to select and view several items at once. See more layout guidelines on the [form page](/web-components/form/).
```
Comment (mathisscott) [inline apps/website/core-components/select/README.md]: `Select elements are` not `Select is`

`Select elements with the multiple attribute` not `The select multiple option`

`enables the user` not `enables to user`

File: apps/website/core-components/toggle/README.md [hunk 6]
```diff
@@ -81,3 +81,123 @@ Toggle switches take up less space than an “on/off” radio button group and c
...
+
+::: component-section-level-one-title
+
+## Layouts
+
+:::
+
+::: component-section-level-one
+
+Toggle is available in vertical, vertical inline, horizontal, horizontal inline, compact layout. The align options for Toggle are useful for creating continuity with other UI elements that are related to the toggle selection. See more layout guidelines on the form page. [form page](/web-components/form/).
```
Comment (mathisscott) [inline apps/website/core-components/toggle/README.md]: `horizontal, horizontal inline, and compact layout` not `horizontal, horizontal inline, compact layout`

**Review by mathisscott — CHANGES_REQUESTED** [review]

**Review by mathisscott — APPROVED** [review]

**Pull request merged**